portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java

```diff
@@ -32,8 +38,23 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     long id = Long.parseLong(request.getParameter("id"));
 
-    Key commentEntityKey = KeyFactory.createKey("Comment", id);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.delete(commentEntityKey);
+    // if we have the id of a particular comment
+    if (id != -1) {
+        Key commentEntityKey = KeyFactory.createKey("Comment", id);
+        datastore.delete(commentEntityKey);
+    } 
+    // will delete all comments
+    else { 
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style says `there is no line break after the brace if it is followed by else or a comma.`
```suggestion
    } else {
        // Will delete all comments 
```